scripts/live_test/CLITest.yml

```diff
@@ -176,16 +186,35 @@ jobs:
         pip install pytest-json-report
         pip install pytest-html
         pip install pytest-rerunfailures
+
         azdev setup -c azure-cli
+
         # az login --service-principal --username $(APP_ID) --password $(PASSWORD) --tenant $(TENANT_ID)
         az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
         az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
-        # Sequential
-        azdev test $(Target) $(USER_LIVE) --mark serial --xml-path test_results.sequential.xml --no-exitfirst -a "-n 1 --json-report --json-report-summary --json-report-file=$(Target).report.sequential.json --html=$(Target).report.sequential.html --self-contained-html --reruns 3 -s"
-        # Parallel
-        azdev test $(Target) $(USER_LIVE) --mark "not serial" --xml-path test_results.parallel.xml --no-exitfirst -a "-n $(USER_PARALLELISM) --json-report --json-report-summary --json-report-file=$(Target).report.parallel.json --html=$(Target).report.parallel.html --self-contained-html --reruns 3 -s"
+
+        # Commit changes
+        if [[ "$(USER_USERNAME)" != "" || "$(USER_TOKEN)" != "" ]]; then
+          azdev test $(Target) --no-exitfirst -a "-n $(USER_PARALLELISM)"
```

Comment:
Why is `Target` not capitalized like `USER_USERNAME`?

Reply:
one is internal, one is user variable.